lib/std/special/build_runner.zig

```diff
@@ -112,6 +112,8 @@ pub fn main() !void {
                 builder.verbose_cc = true;
             } else if (mem.eql(u8, arg, "--verbose-llvm-cpu-features")) {
                 builder.verbose_llvm_cpu_features = true;
+            } else if (mem.eql(u8, arg, "--system-linker-hack")) {
+                builder.system_linker_hack = true;
```

Comment:
I think we should keep it local to each artifact - no changes to this file needed.